### ISSUE TITLE
EmptyLinesAroundArguments: False positive

### ISSUE THREAD

**Issue by jaredbeck** [rubocop-hq/rubocop#9097]
## Input

```ruby
# frozen_string_literal: true

def execute
  ::Redacted.
    redacted(@redacted).
    redacted2(@redacted2).
    redacted3(redacted).

    # redacted
    # redacted
    # blank lines help the reader understand that these comments pertain only to
    # the one following method call (`references`)
    # redacted
    # redacted
    references(:redacted, :redacted2, :redacted3, :redacted4, :redacted5).

    redacted4(redacted2).
    redacted5(@redacted3).
    redacted6(redacted: true)
end
```

## Expected behavior

No offenses expected

## Actual behavior

```
bin/rubocop input.rb
Inspecting 1 file
C

Offenses:

input.rb:16:1: C: Layout/EmptyLinesAroundArguments: Empty line detected around arguments.

1 file inspected, 1 offense detected, 1 offense auto-correctable
```

## Steps to reproduce the problem

See input above.

## RuboCop version

```
bundle exec rubocop -V
1.4.2 (using Parser 2.7.2.0, rubocop-ast 1.2.0, running on ruby 2.6.6 x86_64-darwin19)
  - rubocop-rails 2.8.1
  - rubocop-rspec 2.0.0
```
